modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-admin/dataSets.spec.ts

```diff
@@ -200,6 +208,26 @@ test('Create parameterized data set via UI', async ({dataSetsPage, page}) => {
 	});
 });
 
+test(
+	'Create parameterized data set with resolved parameters via UI',
+	{tag: '@LPD-25230'},
+	async ({dataSetsPage, page}) => {
+		await test.step('Create Data Set', async () => {
+			await dataSetsPage.goto();
+			await dataSetsPage.createDataSet(cartDataSetConfig);
+		});
+
+		await assertTableColumnLabels(page);
+
+		await assertTableCellContent({dataSetConfig: cartDataSetConfig, page});
+
+		await assertTableActionLabels(page);
+
+		await test.step('Delete Data Set', async () => {
+			await dataSetsPage.deleteDataSet(blogPostsDataSetConfig.name);
```

Comment:
> as for data-set-web the whole suite always goes timeout in different points and in local it's not verifiable.

This might have been a global issue fixed in https://github.com/liferay-frontend/liferay-portal/pull/4369/commits/dd8254e7f52070558628f9c38e1da50a68a83143